content/rs/administering/designing-production/security/hardening.md

```diff
@@ -0,0 +1,39 @@
+---
+Title: Deployment Hardening 
+description:
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+---
+
+
+## Deployment Hardening
```

Comment:
@Paz-Yanover Seems OK to me but I would call it Disabling commands and put it in https://docs.redislabs.com/latest/rs/administering/designing-production/security/

I think the point of this article is really what we as Redis Labs recommend that you disable to prevent improper use of commands.